checkov/github_actions/checks/job/CosignSBOM.py

```diff
@@ -22,7 +22,7 @@ def __init__(self) -> None:
         )
 
     def scan_entity_conf(self, conf: dict[str, Any]) -> tuple[CheckResult, dict[str, Any]]:
-        if not conf:
+        if not conf or not isinstance(conf, dict):
```

Comment:
```suggestion
        if not isinstance(conf, dict):
```
Since `None` is not `dict`